invenio_drafts_resources/services/records/components/files.py

```diff
@@ -81,13 +87,15 @@ def update_draft(self, identity, data=None, record=None, errors=None):
         try:
             self.files_component.assign_files_enabled(draft, enabled)
         except ValidationError as e:
-            errors.append({"field": "files.enabled", "messages": e.messages})
+            errors.append(
+                {"field": f"{self.files_data_key}.enabled", "messages": e.messages}
+            )
             return  # exit early
 
-        if draft.files.enabled and not draft.files.items():
+        if draft_files.enabled and not draft_files.items():
             errors.append(
                 {
-                    "field": "files.enabled",
+                    "field": f"{self.files_data_key}.enabled",
```

Comment:
Question: How do we know is `data_key` and not `attribute_key`? they are potentially different (according to `FilesOptionsComponent`. However, do we have a way to enforce that generators will use `data_key`?